node/pkg/flowlogs/flowlogs.go

```diff
@@ -0,0 +1,89 @@
+// Copyright (c) 2025 Tigera, Inc. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//	http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package flowlogs
+
+import (
+	"context"
+	"fmt"
+	"os/signal"
+	"syscall"
+	"time"
+
+	"github.com/sirupsen/logrus"
+
+	"github.com/projectcalico/calico/felix/collector/goldmane"
+	"github.com/projectcalico/calico/goldmane/pkg/types"
+	"github.com/projectcalico/calico/goldmane/proto"
+)
+
+func StartAndWatch(num int) {
+	ctx, stop := signal.NotifyContext(context.Background(), syscall.SIGINT, syscall.SIGTERM)
+	defer stop()
+
+	nodeServer := goldmane.NewNodeServer(goldmane.NodeSocketDir)
+	err := nodeServer.Run()
+	if err != nil {
+		logrus.WithError(err).Error("Failed to start goldmane node server")
+		return
+	}
+	defer nodeServer.Stop()
+
+	nodeServer.Watch(ctx, num, time.Second, func(flow *types.Flow) {
```

Comment:
I think it's going to be important to have an option to emit the raw JSON logs here to get the full structure. WDYT?

Reply:
Definitely we should. As you mentioned, we should use a proper CLI and add different options.